content/en/docs/contributing/release-process.md

```diff
@@ -102,34 +128,54 @@ The process for cutting a minor release is as follows:
 
         You can view the progress by clicking the Google Cloud Build URL in the output of this command.
 
-   2. Next publish the release artifacts for real.
+   2. While the build is running, send a third Slack message in reply to
+      the first message:
 
-        If the last step succeeded, you can now re-run the `cmrel publish` with the `--nomock` argument to actually publish the release artifacts to GitHub, `Quay.io`, [Helm Hub] etc.
+      > Publish dry-run step: <https://console.cloud.google.com/cloud-build/builds/16f6f875-0a23-4fff-b24d-3de0af207463?project=1021342095237>
 
-       ```bash
-       cmrel publish --nomock --release-name <RELEASE_NAME>
-       ```
+   3. Next publish the release artifacts for real.
+
+      If the last step succeeded, you can now re-run the `cmrel publish`
+      with the `--nomock` argument to actually publish the release
+      artifacts to GitHub, `Quay.io`, [Helm Hub] etc.
+
+      ```bash
+      cmrel publish --nomock --release-name <RELEASE_NAME>
+      ```
+
+      NOTE: At this stage there will be a draft release on GitHub and a
+      live release on [Helm Hub]. So you must now complete the release
+      process quickly otherwise users of the latest release on[Helm Hub]
```

Comment:
[Helm Hub] - I think there is a broken/missing link, probably should be pointing to `https://artifacthub.io/packages/helm/jetstack/cert-manager` ? Also the name seems to be ArtifactHub now.

Reply:
Yeah, https://charts.jetstack.io returns a 404 instead of 301 to https://artifacthub.io/packages/helm/jetstack/cert-manager... I investigated in https://github.com/jetstack/cert-manager/issues/3439#issuecomment-768898542 

Can I rename "Helm Hub" to "Helm chart"?